src/components/BookList.js

```diff
@@ -0,0 +1,15 @@
+// import React from 'react';
+// import Book from './Book';
+
+// function BookList() {
+//   return (
+//     <div>
+//       <h2>Book List</h2>
+//       <div>
+//         <Book title={title} author={author} onClick={handelDelete} />
+//       </div>
+//     </div>
+//   );
+// }
+
+// export default BookList;
```

Comment:
- [ ] It is unprofessional to leave commented code in your code base so I kindly suggest you remove them 👍

Reply:
Hi @Efiamotu-1, Sorry I'll remove the unused code